src/test/scala/tech/cryptonomic/conseil/tezos/InMemoryDatabase.scala

```diff
@@ -87,8 +79,9 @@ trait InMemoryDatabase extends BeforeAndAfterAll with BeforeAndAfterEach {
   }
 
   override protected def beforeEach(): Unit = {
-    Await.ready(dbHandler.run(truncateAll), 1.second)
     super.beforeEach()
+    Await.ready(dbHandler.run(truncateAll), 1.second)
+    ()
```

Comment:
Isn't this line useless?

Reply:
Essentially yes, but it removes a warning about discarding non-unit values